Cargo.toml

```diff
@@ -14,7 +14,7 @@ license = "MIT OR Apache-2.0"
 name = "derive-where"
 readme = "README.md"
 repository = "https://github.com/ModProg/derive-where"
-rust-version = "1.34.2"
+rust-version = "1.34"
```

Comment:
I noticed that skipping the minor version seems to very common in other crates right now, it looks reasonable to me and is more correct anyway. 1.34.2 doesn't actually fix anything that concerns us.

Reply:
sounds good